addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2331,6 +2346,43 @@ export class OdooEditor extends EventTarget {
                 }
             }
         } else if (node.nodeType !== Node.TEXT_NODE) {
+            if (node.nodeName === 'TD' && (node.hasAttribute('style') || node.hasAttribute('bgcolor'))) {
+                // TD tags do not support style in v15, move style to child or new SPAN.
+                const keepChild = (node.childNodes.length === 1 && node.childNodes[0].nodeName === 'SPAN') 
+                const styleElem = keepChild ? node.childNodes[0] : this.document.createElement('SPAN');
+                if (node.hasAttribute('bgcolor') && !styleElem.style.cssText.includes('background-color:')) {
+                    // bgcolor attribute is deprecated, replaced with background-color.
+                    styleElem.setAttribute('style', 
+                        `${styleElem.getAttribute('style') || ''};background-color:${node.getAttribute('bgcolor')}`);
+                }
+                styleElem.setAttribute('style', `${styleElem.getAttribute('style') || ''};${node.getAttribute('style') || ''}`);
+                // Give TD's children their new parent.
+                if (!keepChild) {
+                    [...node.childNodes].forEach(child => styleElem.appendChild(child));
+                    node.appendChild(styleElem);
+                }
+                this._cleanForPaste(styleElem);
+            } else if (node.nodeName === 'FONT') {
+                // FONT tags have some style information in custom attributes,
+                // this maps them to the style attribute.
+                const attributeMap = { 'color': 'color', 'size': 'font-size'};
```

Comment:
About the variable name: I can see that it's some kind of map but I don't know what is mapped to what without the comment above. I suggest renaming this variable to `attributesToStyle`.